src/lib/io/include/sol-efivarfs-storage.h

```diff
@@ -252,6 +430,21 @@ sol_efivars_read_string(const char *name, char **value)
     return 0;
 }
 
+/**
+ * @brief Writes a string into an EFI variable.
+ *
+ * This function uses sol_efivars_write_raw() internally, the same
+ * behaviour should be considered.
+ *
+ * @param name name of property. It will create a new EFI variable with
+ * this name.
+ * @param value The value to be written.
+ * @param cb callback to be called when writing finishes. It contains status
+ * of writing: if failed, is lesser than zero.
+ * @param data user data to be sent to callback @a cb
+ *
+ * return @c 0 on success, a negative number on failure
+ */
```

Comment:
It seems good.
I'm wondering if there isn't any way to enforce inline functions documentation on doxygen / doxy-coverage,
otherwise we may let some functions behind.